tests/utility_scripts/test_utility_scripts.py

```diff
@@ -151,3 +170,17 @@ def test_fail_verify_bottom_constant(n, stack):
     context = Context(script=unlock + lock)
 
     assert not context.evaluate()
+
+
+@pytest.mark.parametrize(
+    ("stack_element", "moving_function", "start_index", "end_index", "msg"),
+    [
+        (StackNumber(1, False), roll, 0, 2, r"Moving more elements than self: Self has \d+ elements, end_index: \d+"),
+        (StackNumber(1, False), roll, -1, 2, r"Moving more elements than self: Self has \d+ elements, end_index: \d+"),
```

Comment:
Shouldn't this return Start index must be positive? I guess it should be the first thing checked by the function move

Reply:
Agreed, changed.